### ISSUE TITLE
Failed displaying polar regions

### ISSUE THREAD

**Issue by yonda-yonda** [openlayers/openlayers#15233]
**Describe the bug**
At [examples/reprojection.html](https://openlayers.org/en/latest/examples/reprojection.html), View projection at EPSG:3413 is not working.

The error in console.
```
transform.js:103 Uncaught TypeError: Cannot convert undefined or null to object
    at Ct (transform.js:103:12)
    at Mt (core.js:8:24)
    at inverse (core.js:77:16)
    at proj.js:721:21
    at proj.js:341:23
    at e.Z.transformInv_ (Triangulation.js:91:34)
    at new e.Z (Triangulation.js:153:36)
    at f (Tile.js:173:27)
    at s.getTile (TileImage.js:322:21)
    at g.getTile (TileLayer.js:128:27)
```  

versions
```json
 {
    "ol": "8.1.0",
    "proj4": "2.9.0"
 }
```

**To Reproduce**
1. Go to [examples/reprojection.html](https://openlayers.org/en/latest/examples/reprojection.html).
2. Click on `NSIDC Polar Stereographic North (EPSG:3413)` of View projection's options.
3. See console.

**Expected behavior**
There seems to be an error in transform since v2.8.1 of proj4.
I don't have idea how to fix it, but is there any way to revert the version back to 2.8.0, or catch proj4 errors, etc.?

**Comment by mike-000** [comment]
3413 works in https://openlayers.org/en/latest/examples/reprojection-by-code.html where proj4 uses WKT definitions.  If you need a workaround replacing the proj4 string with

`PROJCS["WGS 84 / NSIDC Sea Ice Polar Stereographic North",GEOGCS["WGS 84",DATUM["WGS_1984",SPHEROID["WGS 84",6378137,298.257223563,AUTHORITY["EPSG","7030"]],AUTHORITY["EPSG","6326"]],PRIMEM["Greenwich",0,AUTHORITY["EPSG","8901"]],UNIT["degree",0.0174532925199433,AUTHORITY["EPSG","9122"]],AUTHORITY["EPSG","4326"]],PROJECTION["Polar_Stereographic"],PARAMETER["latitude_of_origin",70],PARAMETER["central_meridian",-45],PARAMETER["false_easting",0],PARAMETER["false_northing",0],UNIT["metre",1,AUTHORITY["EPSG","9001"]],AUTHORITY["EPSG","3413"]]`

also works in this example.

**Comment by ahocevar** [comment]
This was caused by a bug in proj4js, which has been fixed in version 2.9.1. Thanks @MoonE for the help with this!
